dir.traversal.targets

```diff
@@ -28,33 +28,29 @@
   </Target>
 
   <!-- build vertical requires the FindBestConfiguration task which is currently in CoreFx.Tools.dll -->
-  <Import Project="buildvertical.targets" Condition="Exists('$(CoreFxToolsTaskDir)CoreFx.Tools.dll')" />
+<!-->  <Import Project="buildvertical.targets" Condition="Exists('$(CoreFxToolsTaskDir)CoreFx.Tools.dll')" /> -->
 
   <Target Name="BuildAllProjects">
     <PropertyGroup>
       <DefaultBuildAllTarget Condition="'$(DefaultBuildAllTarget)'==''">$(MSBuildProjectDefaultTargets)</DefaultBuildAllTarget>
     </PropertyGroup>
 
-    <MSBuild Targets="AnnotateProjectsWithConfiguration"
```

Comment:
I don't think we should turn the whole thing off just to fix packages.builds.  The projects it builds do their own filtering.  If you'd like me to get this fixed I can do that but don't sacrifice progress here.